### PR TITLE
Add an example for Bazel's Docker sandbox mode

### PR DESCRIPTION
**Pull request by laszlocsomor**
The shell script creates a sample workspace with
all necessary files to demonstrate this feature.

### CODE REVIEW

File: docker-sandbox/create-workspace.sh [hunk 1]
```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
```
Comment (ulfjack) [inline docker-sandbox/create-workspace.sh:1]: Instead of a shell script, we could just check in the files here?

Comment (laszlocsomor) [reply docker-sandbox/create-workspace.sh:1]: We could, but then you'd have to edit the docker image in two different files :(

Comment (ulfjack) [reply docker-sandbox/create-workspace.sh:1]: why would you change the docker image?

Comment (ulfjack) [reply docker-sandbox/create-workspace.sh:1]: (Also, we can change Bazel to use the container-image property. That should be pretty quick to get merged.)

Comment (laszlocsomor) [reply docker-sandbox/create-workspace.sh]: Done.

**Review by ulfjack — APPROVED** [review]

**Pull request merged**